support-frontend/app/services/stepfunctions/SupportWorkersClient.scala

```diff
@@ -173,11 +160,11 @@ class SupportWorkersClient(
         ipAddress =
           request.headers.get("X-Forwarded-For").flatMap(_.split(',').headOption).getOrElse(request.remoteAddress),
       )
-      isExistingAccount = createPaymentMethodState.paymentFields.left.exists(_.isInstanceOf[ExistingPaymentFields])
+      isExistingAccount = createPaymentMethodState.paymentFields.isInstanceOf[ExistingPaymentFields]
```

Comment:
this code probaly predates any of this, but annoyingly the simplest way - `isInstanceOf` - is not type safe as it only gives a warning if the types are not compatible.  You would have to either use a pattern match or a partial function to get the type safety.
https://scastie.scala-lang.org/xG1JBBG0QcW9ehKDG05jMA

Partial function is probably less verbose than pattern match
```scala
PartialFunction.cond(createPaymentMethodState.paymentFields) { case _: ExistingPaymentFields => true }
```